libsawtooth/src/state/merkle.rs

```diff
@@ -678,7 +676,7 @@ fn encode_and_hash(node: Node) -> Result<(Vec<u8>, Vec<u8>), StateDatabaseError>
 /// Given a path, split it into its parent's path and the specific branch for
 /// this path, such that the following assertion is true:
 ///
-/// ```
+/// ```ignore
```

Comment:
This isn't captured in the commit message; why is this necessary?

Reply:
added to commit, this test was not being running in its old location but when moved was being ran. But the doc test fails, so I ignored it.